src/hadronization/HybridHadronization.cc

```diff
@@ -573,13 +573,17 @@ void HybridHadronization::DoHadronization(vector<vector<shared_ptr<Parton>>>& sh
       surface.emplace_back(std::move(cell));
     }
 
+    auto start = std::chrono::high_resolution_clock::now();
```

Comment:
The timing should not propagate into the final version.

Reply:
I removed these time commands.  Thanks.